contrib/vexecutor/execVScan.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "execVScan.h"
+#include "miscadmin.h"
+
+static TupleTableSlot*
+ExecVScan(ScanState *node, ExecScanAccessMtd accessMtd);
+
+static const ScanMethod *
+getVScanMethod(int tableType)
+{
+    static const ScanMethod scanMethods[] =
+            {
+                    //HEAPSCAN
+                    {
+                    },
+                    //APPENDONLYSCAN
+                    {
+                            &AppendOnlyScanNext, &BeginScanAppendOnlyRelation, &EndScanAppendOnlyRelation,
```

Comment:
Are these scan functions placeholders?

Reply:
Well, this PR only provide a vectorized scan framework, AO and Parquet vectorized scan function will implement later. For easy to do feature test for this PR. I use normal AO and Parquet function as the tuple scanner under the framework